repairnator/kubernetes-support/repairnator-deployment-yamlfiles/repairnator-scanner.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: repairnator-scanner
+  labels:
+    app: repairnator-scanner
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: repairnator-scanner
+  template:
+    metadata:
+      labels:
+        app: repairnator-scanner
+    spec:
+      containers:
+      - name: repairnator-scanner
+        image: tailp/repairnator-scanner
```

Comment:
Don't forget to update the image name when you have pushed the image to the repairnator account.

Reply:
yes I'll change that :) .